parlai/crowdsourcing/tasks/turn_annotations/run.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+from dataclasses import dataclass, field
+from typing import Any, List
+
+import hydra
+from mephisto.operations.hydra_config import register_script_config
+from omegaconf import DictConfig
+
+from parlai.crowdsourcing.tasks.turn_annotations.turn_annotations_blueprint import (
+    BLUEPRINT_TYPE,
+)
+from parlai.crowdsourcing.tasks.turn_annotations.impl import run_task
+from parlai.crowdsourcing.utils.mturk import MTurkRunScriptConfig
+
+
+TASK_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
+
+
+defaults = [
+    {'mephisto/blueprint': BLUEPRINT_TYPE},
+    {"mephisto/architect": "local"},
+    {"mephisto/provider": "mock"},
+    {"conf": "example"},
+]
+
+
+@dataclass
+class ScriptConfig(MTurkRunScriptConfig):
+    defaults: List[Any] = field(default_factory=lambda: defaults)
+    task_dir: str = TASK_DIRECTORY
+    monitoring_log_rate: int = field(
+        default=30,
+        metadata={
+            'help': 'Frequency in seconds of logging the monitoring of the crowdsourcing task'
+        },
+    )
+
+
+register_script_config(name='scriptconfig', module=ScriptConfig)
+
+
+@hydra.main(config_name="scriptconfig")
+def main(cfg: DictConfig) -> None:
```

Comment:
Hydra 101 question: using this can we have three different configs and the task can be launched via the command line with just setting a param to point to the different configs?

Reply:
Yeah, you can specify a new config on the command line as described in https://github.com/facebookresearch/ParlAI/tree/master/parlai/crowdsourcing/tasks#specifying-your-own-yaml-file